plugins/lowering/src/main/java/org/qbicc/plugin/lowering/Lowering.java

```diff
@@ -61,7 +65,7 @@ public GlobalVariableElement getGlobalForField(FieldElement fieldElement) {
         builder.setEnclosingType(enclosingType);
         final String sectionName;
         if (fieldElement.getType() instanceof ReferenceType) {
-            sectionName = GLOBAL_REFERENCES;
+            sectionName = CompilationContext.IMPLICIT_SECTION_NAME; // TODO: GLOBAL_REFERENCES;
```

Comment:
This is a hack around a mach os problem with the way sections are named. I think the section needs to be named DATA,QBICC_GLOBALS when targeting mach os.